code/game/objects/items/stacks/medical.dm

```diff
@@ -42,9 +42,9 @@
 				user << "<span class='warning'>[H] isn't bleeding!</span>"
 				return
 
-	if(isliving(M))
-		if(!M.can_inject(user, 1))
-			return
+	// if(isliving(M))
```

Comment:
Just a quick question, this isn't the code that stops syringes shot from syringe guns from penetrating suits, is it?